dm/simulator/config/config.go

```diff
@@ -29,3 +36,32 @@ type ColumnDefinition struct {
 	DataType   string `yaml:"type"`
 	DataLen    int    `yaml:"length"`
 }
+
+func (t *TableConfig) GenCreateTable() string {
+	var buf strings.Builder
+	buf.WriteString("CREATE TABLE ")
+	buf.WriteString(dbutil.TableName(t.DatabaseName, t.TableName))
+	buf.WriteByte('(')
+	for i, col := range t.Columns {
+		if i != 0 {
+			buf.WriteByte(',')
+		}
+		buf.WriteString(dbutil.ColumnName(col.ColumnName))
+		buf.WriteByte(' ')
+		buf.WriteString(col.DataType)
+		if col.DataLen > 0 {
+			buf.WriteByte('(')
+			buf.WriteString(strconv.Itoa(col.DataLen))
+			buf.WriteByte(')')
+		}
+	}
+	for _, ukColName := range t.UniqueKeyColumnNames {
```

Comment:
not sure `UniqueKeyColumnNames` represents multiple columns in one UK or multiple UK which has one column

Reply:
It's multiple columns in one UK, fix it.